src/lib/functions/synchronous.js

```diff
@@ -1,55 +1,37 @@
 const { Buffer } = require('buffer')
 
-const lambdaLocal = require('lambda-local')
-
 const { NETLIFYDEVERR } = require('../../utils/logo')
 
-const { detectAwsSdkError, DEFAULT_LAMBDA_OPTIONS, SECONDS_TO_MILLISECONDS } = require('./utils')
-
-const createSynchronousFunctionCallback = function ({ response, warn }) {
-  return function callbackHandler(err, lambdaResponse) {
-    if (err) {
-      return handleErr({ error: err, response, warn })
-    }
+const handleSynchronousFunction = function (err, result, response) {
```

Comment:
Similarly to `src/lib/functions/background.js`, this is now runtime-agnostic. This method simply serves the result of a synchronous function, taking its result and terminating the response.